app/src/main/java/fr/free/nrw/commons/explore/recentsearches/RecentSearchesFragment.java

```diff
@@ -45,32 +45,32 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         ButterKnife.bind(this, rootView);
         recentSearches = recentSearchesDao.recentSearches(10);
 
-      if(recentSearches.isEmpty()) {
+        if (recentSearches.isEmpty()) {
```

Comment:
Please submit unrelated whitespace fixes as a separate PR (feel free to request review from me for this)

Reply:
I just summit reformatted code